app/routes/index.tsx

```diff
@@ -1,11 +1,52 @@
-import { LinksFunction } from '@remix-run/node';
+import { LinksFunction, LoaderFunction } from '@remix-run/node';
 import styles from 'react-datepicker/dist/react-datepicker.css';
 import Calendar from '~/components/Calendar';
 import Navbar from '~/components/common/navbar';
+import axios from 'axios';
+import moment from 'moment';
+import { useStore } from '~/store/useStore';
+import { useLoaderData } from '@remix-run/react';
+import { parseEvents } from '~/utils/event.utils';
+import { getEvents } from '~/constants/urls.constants';
+import { ToastContainer, toast } from 'react-toastify';
+import { useEffect } from 'react';
```

Comment:
the order of imports should be 
1. imports from libs/packages
2. imports from local directory
We should expect es-lint to give warnings for this,

Reply:
I agree, eslint is not working as expected. It can be taken up in the eslint fix as you mentioned. Resolving this conversation.